tabs/pid_tuning.css

```diff
@@ -287,7 +302,7 @@
 }
 
 .tab-pid_tuning table td {
-    padding: 1px;
+    padding: 5px;
```

Comment:
pls redo this edit ... makes more new probs

![padding](https://cloud.githubusercontent.com/assets/20139767/18428303/71954b16-78cd-11e6-9d69-d0b2d9523abb.jpg)